onnxruntime/test/shared_lib/utils.cc

```diff
@@ -27,9 +31,10 @@ void MyCustomKernel::Compute(OrtKernelContext* context) {
 
   // Do computation
 #ifdef USE_CUDA
-  cuda_add(size, out, X, Y);
-  cudaStreamSynchronize(nullptr);
```

Comment:
No need this costly sync anymore - we are launching the custom op in the same stream as ORT's CUDA kernels - previously without this sync, we would have gotten result mismatches as things were not synchronized correctly

Reply:
Can we leave the sync here but commented out saying this should never be necessary if things are setup correctly?

---
In reply to: [592109259](https://github.com/microsoft/onnxruntime/pull/6971#discussion_r592109259) [](ancestors = 592109259)

Reply:
done